client/views.py

```diff
@@ -133,12 +136,18 @@ def schedule(request):
 
 
 def find_certificate_view(request):
-    context = {'form': GetCertificateForm()}
+    context = {}
+    form = GetCertificateForm()
     if request.method == 'POST':
         form = GetCertificateForm(request.POST)
         if form.is_valid():
             # TODO: change to 'get' instead of 'first' after fixing #50
-            volunteer = Volunteer.objects.filter(tz_number=form['id_number'].data).first()
+            volunteers_qs = Volunteer.objects.all()
+            volunteers_qs = volunteers_qs.annotate(calc_full_name=Concat(F('first_name'), Value(' '), F('last_name'),
+                                                                         output_field=CharField()))
```

Comment:
There's the full_name property that does this. Can you use it with annotate?

_this would probably be the same solution to miki's problem with annotate_

Reply:
Since full_name is deprecated and is not populated anymore, using it by itself is not an option. Overriding it with annotate causes issues for now.
When the full name field will be removed, we could change this name as part of a cleanup.